projects/ng-helpers/package.json

```diff
@@ -18,8 +18,8 @@
   },
   "homepage": "https://jaspero.co",
   "peerDependencies": {
-    "@angular/common": "^13.0.0",
-    "@angular/core": "^13.0.0"
+    "@angular/common": "14.2.4",
+    "@angular/core": "14.2.4"
```

Comment:
@svendjanis I think here we can just do `^14.0.0`. This is to say the library is compatible with any version of angular v14+